docs/reticulate.rst

```diff
@@ -0,0 +1,517 @@
+    print(fileEntity)
+
+    # view the file on the web
+    syn$onweb(synId)
+
+Create folder and add files to the folder:
+
+  .. code-block::
+
+    dataFolder <- synapseclient$Folder("Data", parent = project)
+    dataFolder <- syn$store(dataFolder)
+
+    filePath <- tempfile()
+    connection <- file(filePath)
+    writeChar("this is the content of the file", connection, eos = NULL)
+    close(connection)
+    file <- synapseclient$File(path = filePath, parent = dataFolder)
+    file <- syn$store(file)
+
+
+Annotating Synapse Entities
+===========================
+
+This illustrates adding annotations to a Synapse entity.
+
+  .. code-block::
+
+    # first retrieve the existing annotations object
+    annotations = syn$get_annotations(project)
+
+    annotations$foo <- "bar"
+    annotations$fooList <- list("bar", "baz")
+
+    syn$set_annotations(annotations)
+
+See `here <index.html#annotating-synapse-entities>`__ for more information on annotations.
+
+Provenance
+==========
+
+This example illustrates creating an entity with associated provenance.
+
+See `here <index.html#provenance>`__ for more information on Provenance related APIs.
+
+  .. code-block::
+
+    act <- synapseclient$Activity(
+      name = "clustering",
+      description = "whizzy clustering",
+      used = c("syn1234", "syn1235"),
+      executed = "syn4567")
+
+  .. code-block::
+
+    filePath <- tempfile()
+    connection <- file(filePath)
+    writeChar("some test", connection, eos = NULL)
+    close(connection)
+
+    file = synapseclient$File(filePath, name="provenance_file.txt", parent=project)
+    file <- syn$store(, activity = act)
+
+
+Tables
+======
+
+These examples illustrate manipulating Synapse Tables.
+Note that you must have installed the Pandas dependency into the Python environment as described
+above in order to use this feature.
+
+See `here <index.html#tables>`__ for more information on tables.
+
+The following illustrates building a table from an R data frame. The schema will be generated
+from the data types of the values within the data frame.
+
+  .. code-block::
+
+    # start with an R data frame
+    genes <- data.frame(
+      Name = c("foo", "arg", "zap", "bah", "bnk", "xyz"),
+      Chromosome = c(1, 2, 2, 1, 1, 1),
+      Start = c(12345, 20001, 30033, 40444, 51234, 61234),
+      End = c(126000, 20200, 30999, 41444, 54567, 68686),
+      Strand = c("+", "+", "-", "-", "+", "+"),
+      TranscriptionFactor = c(F, F, F, F, T, F))
+
+    # build a Synapse table from the data frame.
+    # a schema is automatically generated
+    # note that reticulate will automatically convert from an R data frame to Pandas
+    table <- synapseclient$build_table("My Favorite Genes", project, genes)
+
+    table <- syn$store(table)
+
+Alternately the schema can be specified. At this time when using date values it is necessary
+to use string or millisecond values and explicitly specify the type in the schema due to how
```

Comment:
by "string or millisecond values" did you mean "millisecond values either as string or integer"?